### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by svenwltr**
@rebuy-de/it-platform Release when everybody is convinced, that everything is fine. So this PR may pending a some time.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -18,6 +18,7 @@
 
         <junit.version>4.11</junit.version>
         <consul-api.version>1.1.8</consul-api.version>
+        <mockito.version>2.0.31-beta</mockito.version>
```
Comment (ghost) [inline pom.xml]: do we really need the beta?

Comment (svenwltr) [inline pom.xml]: I really don't get their release policy: http://search.maven.org/#search|gav|1|g%3A%22org.mockito%22%20AND%20a%3A%22mockito-core%22

Comment (florian-zeidler) [inline pom.xml]: Would be fine for me, it is "just" a testing dependency.

Comment (svenwltr) [inline pom.xml]: Ahh .. now I get it. They are working on version two for ages

File: README.md [hunk 2]
```diff
@@ -8,6 +8,17 @@ Primary responsible person (PRP)
 
 * Björn Häuser, [@bjoernhaeuser](https://github.com/bjoernhaeuser)
```
Comment (florian-zeidler) [inline README.md:10]: Shouldn't Sven also be noted here?

**Comment by florian-zeidler** [comment]
Don't we need a version to set up the silos?

**Comment by svenwltr** [comment]
I am sure we will change the API some day. So we should use proper semver.

**Comment by florian-zeidler** [comment]
Then start with 1.0.0 to get up and running and then make a new release if it does not fit anymore or do we already have a version in our maven repository?

**Comment by svenwltr** [comment]
There is no release in Maven at the moment. I'll change the API like @hendrik-rebuy suggested and make a release afterwards, to be able to use it in the silos.

**Comment by svenwltr** [comment]
@bjoernhaeuser @hendrik-rebuy @das-vinculum Are you fine with the current API? I would release it in this case.

**Comment by bjoernhaeuser** [comment]
API is fine for me :+1:

**Comment by ghost** [comment]
:+1:

**Pull request merged**